tests/test_utils.py

```diff
@@ -49,3 +49,15 @@ def test_ignores_non_prefixed_values(self):
 
     def test_takes_prefixed_values(self):
         self.assertDictEqual({'test': 1, 'something_else': 'test'}, collect_parameters_from_request({'p_test': 1, 'p_something_else': 'test'}))
+
+
+class TestCleanFromHtmlTags(TestCase):
+    def test_removes_html_tags(self):
+        dirty_string = '<a href="some/site" target="_brank">value</a>'
+        self.assertEqual('value', clean_from_html_tags(dirty_string))
+
+    def test_ignores_number(self):
+        self.assertEqual(3, clean_from_html_tags(3))
+
+    def test_ignores_string_with_no_html_tag(self):
+        self.assertEqual('value', clean_from_html_tags('value'))
```

Comment:
If it raises an error, you might want to look at an alternative way to clean the HTML. I've seen a few very simple code samples on StackOverflow that do it with no external dependencies.